dynamic_instrumentation/src/instrument.rs

```diff
@@ -22,8 +22,8 @@ use crate::arg::{ArgKind, InstrumentationArg};
 use crate::hooks::Hooks;
 use crate::mir_utils::{has_outer_deref, remove_outer_deref, strip_all_deref};
 use crate::point::cast_ptr_to_usize;
-use crate::point::InstrumentationAdder;
 use crate::point::InstrumentationApplier;
+use crate::point::InstrumentationPointCollectorVisitor;
```

Comment:
I hate to nitpick when I'd already commented on this point, but `InstrumentationPointCollectorVisitor` stutters in a way that makes it sound like it's a visitor of `InstrumentationPointCollector`s. I think a verb-like name is ideal, as a visitor class/struct exists for the behavior (verb) in its `Visitor` impl. So I would recommend `CollectInstrumentationPoints` or `CollectInstrumentationPointsVisitor`.

A high-level perspective on this is that the whole visitor is an alternative way to write a `collect_instrumentation_points` function.